src/models/create_token_transaction.ts

```diff
@@ -127,19 +127,23 @@ class CreateTokenTransaction extends Transaction {
     array.push(symbolBytes);
   }
 
-  getTokenInfoFromBytes(buf: Buffer): Buffer {
+  getTokenInfoFromBytes(_buf: Buffer): Buffer {
```

Comment:
Maybe choose another name since underscore indicates that the variable is not used

Reply:
Good point. Renamed in [3eef8f2](https://github.com/HathorNetwork/hathor-wallet-lib/pull/679/commits/3eef8f27f90e242fb2f5de5b75249d1419eb8d85)